aws/src/integration/java/org/apache/iceberg/aws/dynamodb/TestDynamoDbCatalog.java

```diff
@@ -296,6 +296,6 @@ public void testDropNamespace() {
   }
 
   private static String genRandomName() {
-    return UUID.randomUUID().toString().replace("-", "");
+    return "IcebergAwsCI_" + UUID.randomUUID().toString().replace("-", "");
```

Comment:
Nit : for consistency across
```suggestion
    return "iceberg_aws_ci_" + UUID.randomUUID().toString().replace("-", "");
```